execution/gethexec/executionengine.go

```diff
@@ -712,6 +823,7 @@ func (s *ExecutionEngine) digestMessageWithBlockMutex(num arbutil.MessageIndex,
 	if err != nil {
 		return nil, err
 	}
+	s.cacheL1PriceDataOfMsg(num, receipts, block)
```

Comment:
missing one more callsite, in sequenceDelayedMessageWithBlockMutex

Reply:
Based on this message https://github.com/OffchainLabs/nitro/pull/2415#discussion_r1649530286, I thought that we don't want to call cacheL1PriceDataOfMsg from sequenceDelayedMessageWithBlockMutex.

Reply:
Sorry, I misinterpreted https://github.com/OffchainLabs/nitro/pull/2415#discussion_r1649530286e and how Nitro handles delayed messages.
Added a cachedL1PriceData call from sequenceDelayedMessageWithBlockMutex :)